app/models/concerns/duplicate_fixable.rb

```diff
@@ -17,6 +17,8 @@ module DuplicateFixable
     scope :high_duplicates, lambda {
       joins(:duplicate).where('score IS NOT NULL AND score >= ? AND sort <= 500', score_upper_threshold)
     }
+
+    after_commit :maybe_notify_duplicates, on: :create
```

Comment:
I assumed this would be on the `Duplicate` model?

Reply:
updated this